frontend/RaffleBay/RaffleBay/View/SignupView.swift

```diff
@@ -0,0 +1,225 @@
+                            self.value = signupFrameHeight * 3
+                        }
+                        
+                    SecureField("Password Confirmation", text: $passwordComfirmation)
+                        .textFieldStyle(SignUpStyle())
+                        .onTapGesture {
+                            self.value = signupFrameHeight * 4
+                        }
+
+                    TextField("Street Address", text: $streetAddress)
+                        .textFieldStyle(SignUpStyle())
+                        .onTapGesture {
+                            self.value = signupFrameHeight * 5
+                        }
+
+                    TextField("City", text: $city)
+                        .textFieldStyle(SignUpStyle())
+                        .onTapGesture {
+                            self.value = signupFrameHeight * 6
+                        }
+                    
+                    HStack(){
+                        TextField("State", text: $state)
+                        .textFieldStyle(SignUpStyle())
+                        
+                        TextField("Zipcode", text: $zipcode)
+                        .textFieldStyle(SignUpStyle())
+                    }
+                        .onTapGesture {
+                            self.value = signupFrameHeight * 7
+                        }
+                    
+                    TextField("Phone Number", text: $phoneNumber)
+                        .textFieldStyle(SignUpStyle())
+                        .keyboardType(.numberPad)
+                        .onTapGesture {
+                            self.value = signupFrameHeight * 8
+                        }
+                    
+//                    DatePicker("Birthdate", selection: $birthdate, in: ...Date(), displayedComponents: .date)
+//                        .labelsHidden()
+//                        .textFieldStyle(SignUpStyle())
+                        
+                    
+                }
+                
+                Spacer()
+                
+                //Login Stack
+                //ZStack here to allow for custom shadow manipulation.
+                ZStack(){
+                    //Create a Rectangle behind the button to simulate a smaller shadow for a 3D effect.
+                    RoundedRectangle(cornerRadius: 25)
+                    .fill(Color.white)
+                    .frame(minWidth:0, maxWidth: 300, minHeight:10, maxHeight: 10)
+                    .shadow(color: Color("PurpleBlue"), radius: 5.0, x: 0, y: 10)
+                    .offset(y:22)
+                    .opacity(0.6)
+                    
+                    //Signup Button
+                    Button(action: {
+                        
+                    }){
+                        Text("Signup")
+                            .fontWeight(.medium)
+                            .padding(10)
+                            .font(.custom("Poppins", size: 30))
```

Comment:
make a style.swift file that contains all the styling constants like "Poppins" etc